web/src/styles/components/buttons.scss

```diff
@@ -114,3 +114,46 @@
     opacity: 0.8;
   }
 }
+
+.landing-page-primary-tile {
+  display: flex;
+  align-items: center;
+  padding: 8px 16px;
+  gap: 16px;
+  font-weight: bold;
+  background-color: #ecf4fb;
+  width: 264px;
+  height: 68px;
+  border: 2px solid #cfe8ff;
+  border-radius: 10px;
+
+  @media (min-width: 643px) {
+    flex-direction: column;
+    padding: 20px 20px 24px 20px;
+    gap: 18px;
+    width: 200px;
+    height: 155px;
+    border-radius: 20px;
+  }
+}
+
+.landing-page-secondary-tile {
+  display: flex;
+  align-items: center;
+  padding: 8px 16px;
+  gap: 16px;
+  background-color: #ffffff;
+  width: 264px;
+  height: 68px;
+  border: 2px solid #e6e6e6;
+  border-radius: 10px;
+
+  @media (min-width: 643px) {
+    flex-direction: column;
+    padding: 20px 20px 24px 20px;
+    gap: 18px;
+    width: 200px;
+    height: 155px;
+    border-radius: 20px;
+  }
+}
```

Comment:
A couple things here: 
1. We always use rem, not px. This is for accessibility purposes; if you want to learn more you can take a look at [articles like this](https://uxdesign.cc/why-designers-should-move-from-px-to-rem-and-how-to-do-that-in-figma-c0ea23e07a15)
2. We always use variables in our sass files, which ensures that our styleguides are always being followed. In your case, that means replacing any hardcoded hex colors with the appropriate variable name, and also replacing the hardcoded pixel widths with their variable names. These can be found in `variables.scss`
3. There's a ton of repeated / duplicated styles between these two classes. It seems like a better-architected approach would be to have a single class called "landing-page-tile" that defined the styles, and then have classes for "primary" and "secondary" (or even, maybe just "secondary"... "primary" is _primary_ after all...), which would define only the styles that are unique to being "secondary"